packages/snjs/lib/application.ts

```diff
@@ -152,9 +154,11 @@ export class SNApplication {
   private itemManager!: ItemManager;
   private keyRecoveryService!: SNKeyRecoveryService;
   private preferencesService!: SNPreferencesService;
+  private permissionsService!: SNPermissionsService;
+
 
   private eventHandlers: ApplicationObserver[] = [];
-  private services: PureService<any>[] = [];
+  private services: PureService<any, any>[] = [];
```

Comment:
why not `unknown` instead of `any`?

Reply:
With unknown I wouldn't be able to push services into this array without an explicit cast, which I didn't think was much of a better solution… If you've got an idea on how to make this better please let me know

Reply:
I see. No quick solution I can figure out.